scripts/cc-cli/README.md

```diff
@@ -0,0 +1,84 @@
+# Creditcoin CLI Tool
+
+⚠️🔧 Warning: This tool is currently under development! 🔧⚠️
+
+Please be aware that the command line tool you are using is still in active development. It may contain bugs, incomplete features, or unexpected behavior. Exercise caution and use it at your own risk. Feedback and bug reports are greatly appreciated to help improve the tool and ensure its stability.
+
+## Building
+
+This tool depends on `creditcoin-js`. Make sure to pack the library using `yarn pack` in the `creditcoin-js` folder before building. It should be available as `creditcoin-js-v0.9.0.tgz`.
```

Comment:
Note: version number has already changed so maybe not a good idea to hard-code it in the readme.

Reply:
Changed to `creditcoin-js-vX.Y.Z.tgz`